app/code/community/Bolt/Boltpay/Helper/Api.php

```diff
@@ -259,14 +257,19 @@ public function createOrder($reference, $session_quote_id = null)
 
         $quote->setTotalsCollectedFlag(false)->collectTotals()->save();
 
-        $existingOrder = Mage::getModel('sales/order')->loadByIncrementId($display_id);
+        /**
+         * TODO: Investigate if this is a problem.  Logically, it makes no sense that the order
```

Comment:
What is the next step here? If you have clear idea on how to investigate, maybe create asana task and put link here? If not, I suggest we don't add comment.

Reply:
Ok.  The suggestion makes sense and I will factor in the public nature of the code and repository to convey these ideas.  A mix of GitHub, Asana, and Slack could easily be used instead of inline code comments for this sort of dialogue.